Modules/Microsoft365DSC/Examples/Resources/IntuneWindowUpdateForBusinessRingUpdateProfileWindows10/1-IntuneWindowUpdateForBusinessRingUpdateProfileWindows10-Example.psm1

```diff
@@ -1,62 +0,0 @@
-<#
-This example is used to test new resources and showcase the usage of new resources being worked on.
-It is not meant to use as a production baseline.
-#>
-
-Configuration Example
```

Comment:
Shouldn't this file be a .ps1 and not a .psm1?

Reply:
These were renamed from psm1 to ps1. This is the old psm1 file